test/blackbox-tests/test-cases/github717-odoc-index/index.mld

```diff
@@ -1 +1 @@
-Test index.
\ No newline at end of file
+{1 Test index}
```

Comment:
If this is a title, it should probably have level 0.

Reply:
done